crates/symbolicator-native/src/caches/ppdb_caches.rs

```diff
@@ -149,5 +149,12 @@ fn write_ppdb_cache(file: &mut File, object_handle: &ObjectHandle) -> CacheEntry
     let file = writer.into_inner().map_err(io::Error::from)?;
     file.sync_all()?;
 
+    // Parse the ppdbcache file to verify integrity
+    let bv = ByteView::map_file_ref(file)?;
+    if PortablePdbCache::parse(&bv).is_err() {
```

Comment:
Out of curiosity, how expensive is it to parse this again?

Reply:
just a couple of bounds checks. It does not fully validate all the data.